pyprocessing/color.py

```diff
@@ -4,25 +4,40 @@
 
 
 class Color:
-    RGB = 0
-    HSV = 1
+    RGB = 1
     HLS = 2
+    HSB = 3
```

Comment:
now that I think about it, the order should be RGB = 0, HSB = 1 and HSV = 2

Reply:
HSB and HSV are the same thing. HLS isn't in processing and is called HSL everywhere except colorsys, but I've left it in there.
I would agree that the numbers _should_ start from 0, but processing has RGB as 1 and HSB as 3. I am sure that some people use the numbers instead of the constants, and having them start from 0 is only stylistic.